stories/locfeature.IMMER.mdx

```diff
@@ -25,10 +25,181 @@ taxonomy:
 ---
 
 import Carousel from "../overrides/common/embedded-video-carousel";
-import contentArray from './locfeature.IMMER/carousel_content.json';
+import contentArray from './locfeature.IMMER/carousel_content';
```

Comment:
`Carousel ` component doesn't seem to be necessary anymore?

Reply:
Yes, good call. I removed it from the .mdx file.